[All Hosts] (yo office) new overview of the Yeoman generator tool (#3299)
OfficeDev/office-js-docs-pr: docs/excel/custom-functions-batching.md

## View the completed sample

> You can follow this article and paste the code examples into your own project. For example, you can use the [Yeoman generator for Office Add-ins](../develop/yeoman-generator-overview.md) to create a new custom function project for TypeScript, then add all the code from this article to the project. You can then run the code and try it out.

Reviewer: ```suggestion
To view the completed sample, follow this article and paste the code examples into your own project. For example, to create a new custom function project for TypeScript use the [Yeoman generator for Office Add-ins](../develop/yeoman-generator-overview.md), then add all the code from this article to the project. Run the code and try it out.
```

> Also, you can download or view the complete sample project at [Custom function batching pattern](https://github.com/OfficeDev/Office-Add-in-samples/tree/main/Excel-custom-functions/Batching). If you want to view the code in whole before reading any further, take a look at the [script file](https://github.com/OfficeDev/Office-Add-in-samples/blob/main/Excel-custom-functions/Batching/src/functions/functions.js).

Reviewer: ```suggestion
Alternatively, download or view the complete sample project at [Custom function batching pattern](https://github.com/OfficeDev/Office-Add-in-samples/tree/main/Excel-custom-functions/Batching). If you want to view the code in whole before reading any further, take a look at the [script file](https://github.com/OfficeDev/Office-Add-in-samples/blob/main/Excel-custom-functions/Batching/src/functions/functions.js).
```

2. A function to make the remote request when the batch is ready.
3. Server code to respond to the batch request, calculate all of the operation results, and return the values.

> In the following sections you will be shown how to construct the code one example at a time. You'll add each code example to your **functions.ts** file. It's recommended you create a brand new custom functions project using the [Yeoman generator for Office Add-ins](../develop/yeoman-generator-overview.md) generator. To create a new project see [Get started developing Excel custom functions](../quickstarts/excel-custom-functions-quickstart.md) and use TypeScript instead of JavaScript.

Reviewer: ```suggestion
In the following sections you'll learn how to construct the code one example at a time. You'll add each code example to your **functions.ts** file. It's recommended you create a brand-new custom functions project using the [Yeoman generator for Office Add-ins](../develop/yeoman-generator-overview.md) generator. To create a new project, see [Get started developing Excel custom functions](../quickstarts/excel-custom-functions-quickstart.md) and use TypeScript instead of JavaScript.
```
